### ISSUE TITLE
Stopping and starting js9 docker fails

### ISSUE THREAD

**Issue by FastGeert** [Jumpscale/developer#67]
#### Installation information

- jumpscale version: 9.0.3
- operating system: Ubuntu 16.04

#### Detailed description
When trying to start the docker after stopping it, it exits after 6 seconds

#### Steps to reproduce
- Install js9 docker
- Install 0-orchestrator
- Stop docker
- Start docker

**Comment by arahmanhamdy** [comment]
@FastGeert  This issue is related to 0-orchestrator as it exists only after installing it on js9 container because of fail start of  `10_ays9.sh` and `11_orchestrator.sh` from `my_init.d` directory

**Comment by zaibon** [comment]
I re-open cause this line should at least be protected and have a default value in case `HOME` varialbe is not set
```
  File "/opt/code/github/jumpscale/prefab9/JumpScale9Prefab/PrefabCore.py", line 253, in initEnv
    env["HOMEDIR"] = env["HOME"]
KeyError: 'HOME'
```

**Comment by rkhamis** [comment]
Already done here: https://github.com/Jumpscale/prefab9/pull/32
